### PR TITLE
refactor experimental code

### LINKED ISSUE #1308: Replace home page with new playground
Issue to track replacing the home page with new_playground.html

### PR DESCRIPTION
**Pull request by johnpryan**
This removes the old playground and cleans up outdated references like `experimental/` and `NewEmbed`.

fixes #1308

### CODE REVIEW

File: lib/playground.dart [hunk 1]
```diff
@@ -36,124 +46,132 @@ import 'sharing/gist_storage.dart';
...
+      _initSamplesMenu();
+      _initMoreMenu();
+      _initSplitters();
+      _initTabs();
+      _initLayout();
+      _initConsoles();
     });
+  }
 
-    DButton shareButton = DButton(querySelector('#sharebutton'));
```
Comment (RedBrogdon) [inline lib/playground.dart]: Ooh, I can probably rip out some of the old sharing stuff from the gist-related classes now. This was the only place that referenced it, I think.

File: lib/playground.dart [hunk 2]
```diff
@@ -162,246 +180,232 @@ class Playground implements GistContainer, GistController {
...
+    var element = querySelector('#samples-menu');
+
+    var samples = [
+      Sample('215ba63265350c02dfbd586dfd30b8c3', 'Hello World', Layout.dart),
+      Sample('e93b969fed77325db0b848a85f1cf78e', 'Int to Double', Layout.dart),
+      Sample('b60dc2fc7ea49acecb1fd2b57bf9be57', 'Mixins', Layout.dart),
+      Sample('7d78af42d7b0aedfd92f00899f93561b', 'Fibonacci', Layout.dart),
+      Sample('a559420eed617dab7a196b5ea0b64fba', 'Sunflower', Layout.html),
+      Sample('cb9b199b1085873de191e32a1dd5ca4f', 'WebSockets', Layout.html),
+      Sample('b6409e10de32b280b8938aa75364fa7b', 'Counter', Layout.flutter),
```
Comment (RedBrogdon) [inline lib/playground.dart]: The new gists I made this afternoon:

https://gist.github.com/RedBrogdon/ecb28c29c646b7f38139b1e7f44129b7 ("Draggables & physics")
https://gist.github.com/RedBrogdon/40308e0a5f47acba46ba62f4d8be2bf4 ("Implicit animations")

File: lib/playground.dart [hunk 3]
```diff
@@ -795,143 +863,156 @@ class Playground implements GistContainer, GistController {
...
+      editorPanelFooter.clearAttr('hidden');
+      _initOutputPanelTabs();
+      _rightDocPanel.setAttribute('hidden', '');
+      _rightConsoleElement.setAttribute('hidden', '');
+      webTabBar.toggleAttr('hidden', false);
+      webLayoutTabController.selectTab('dart');
+      editorPanelHeader.clearAttr('hidden');
+      webOutputLabel.setAttr('hidden');
+    } else if (layout == Layout.flutter) {
+      _disposeRightSplitter();
```
Comment (RedBrogdon) [inline lib/playground.dart]: This would probably be a good spot to do the Safari check.

File: lib/playground.dart [hunk 4]
```diff
@@ -1,26 +1,36 @@
-// Copyright (c) 2014, the Dart project authors.  Please see the AUTHORS file
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-library playground;
+library new_playground;
```
Comment (RedBrogdon) [inline lib/playground.dart]: Leave as "playground?"

**Comment by johnpryan** [comment]
@RedBrogdon let's follow up with those changes

**Pull request merged**